.github/workflows/ci.yaml

```diff
@@ -119,6 +120,7 @@ jobs:
       with:
         version: '5.15.2'
         cached: ${{ steps.cache-qt.outputs.cache-hit }}
+        setup-python: false
```

Comment:
I guess this will also do it for Linux, but doesn't seem to work

Reply:
and it's fine.
In fact i should have added it everywhere since we don't use PySide at all.